lighthouse-cli/bin.ts

```diff
@@ -44,6 +45,14 @@ interface LighthouseError extends Error {
 
 const cliFlags = getFlags();
 
+if (cliFlags.remoteDevice) {
+  cliFlags.port = defaults(cliFlags.port, 9222);
```

Comment:
Same goes for the other flags, if the user wants to deactivate, say `disableNetworkThrottling`, he/she can do that now.